packages/diagrams/frontend/sirius-components-diagrams/src/renderer/dropNode/useDropNodeStyle.ts

```diff
@@ -36,5 +36,6 @@ export const useDropNodeStyle = (nodeId: string): useDropNodeStyleValue => {
       style.boxShadow = `0px 0px 2px 2px ${theme.palette.primary.main}`;
     }
   }
-  return { style };
+
+  return { style: useMemo(() => style, []) };
```

Comment:
It seems weird to have this memo without dependencies.
By the way, the other nodes during DnD are not faded